### PR TITLE
Change styling of text to accomodate large numbers

### PR DESCRIPTION
**Pull request by JamieTran**
Before:
image

After:
image

### CODE REVIEW

File: src/components/d2l-grade-result-numeric-score.js [hunk 1]
```diff
@@ -1,79 +1,80 @@
...
+			.d2l-grade-result-numeric-score-score {
+				max-width: 5.25rem;
+			}
+			.d2l-grade-result-numeric-score-score-read-only {
+				max-width: 5.25rem;
+				margin-right: 0.5rem;
+			}
+			.d2l-grade-result-numeric-score-score-text {
+				margin-left: 0.75rem;
+				margin-right: 0.25rem;
```
Comment (JamieTran) [inline src/components/d2l-grade-result-numeric-score.js:31]: I'm not sure why it's saying I changed the whole file but this is the only change I made

Comment (jeroach) [reply src/components/d2l-grade-result-numeric-score.js]: Whitespace changes

**Review by jeroach — APPROVED** [review]

File: src/components/d2l-grade-result-numeric-score.js [hunk 2]
```diff
@@ -1,79 +1,80 @@
...
+			}
+			.d2l-grade-result-numeric-score-score {
+				max-width: 5.25rem;
+			}
+			.d2l-grade-result-numeric-score-score-read-only {
+				max-width: 5.25rem;
+				margin-right: 0.5rem;
+			}
+			.d2l-grade-result-numeric-score-score-text {
+				margin-left: 0.5rem;
```
Comment (jeroach) [inline src/components/d2l-grade-result-numeric-score.js:30]: We'll need to account for rtl with these

**Review by jeroach — CHANGES_REQUESTED** [review]
My bad - rtl support needed

File: src/components/d2l-grade-result-numeric-score.js [hunk 3]
```diff
@@ -1,79 +1,94 @@
...
+				margin-right: 0.25rem;
+				text-align: center;
+			}
+			:host([dir="rtl"]) .d2l-grade-result-numeric-score-score-read-only {
+				max-width: 5.25rem;
```
Comment (jeroach) [inline src/components/d2l-grade-result-numeric-score.js:35]: No need to set `max-width`. You also need to zero out the `margin-right`

E.g. `margin-right: 0`

File: src/components/d2l-grade-result-numeric-score.js [hunk 4]
```diff
@@ -1,79 +1,94 @@
...
+				margin-left: 0.5rem;
+			}
+			:host([dir="rtl"]) .d2l-grade-result-numeric-score-score-read-only {
+				max-width: 5.25rem;
+				margin-left: 0.5rem;
+			}
+			:host([dir="rtl"]) .d2l-grade-result-numeric-score-score-text {
+				margin-right: 0.5rem;
+				margin-left: 0.25rem;
+				text-align: center;
```
Comment (jeroach) [inline src/components/d2l-grade-result-numeric-score.js:45]: No need to set `text-align: center`

File: src/components/d2l-grade-result-numeric-score.js [hunk 5]
```diff
@@ -1,79 +1,94 @@
...
+			.d2l-grade-result-numeric-score-score-text {
+				margin-left: 0.5rem;
+				margin-right: 0.25rem;
+				text-align: center;
+			}
+			:host([dir="rtl"]) .d2l-grade-result-numeric-score-score-read-only {
+				max-width: 5.25rem;
+				margin-left: 0.5rem;
+			}
+			:host([dir="rtl"]) .d2l-grade-result-numeric-score-score-read-only {
```
Comment (jeroach) [inline src/components/d2l-grade-result-numeric-score.js:38]: This block looks to be a duplicate.

File: src/components/d2l-grade-result-numeric-score.js (hunk 3 above)
Comment (JamieTran) [reply src/components/d2l-grade-result-numeric-score.js:35]: Do I have to zero out on the normal style as well? `e.g. margin-left: 0rem`

Comment (jeroach) [reply src/components/d2l-grade-result-numeric-score.js:35]: Nope, because the default is 0. Only needed here because a value for ltr was already set

**Review by jeroach — APPROVED** [review]

**Pull request merged**